sdk/storage/Azure.Storage.Blobs.Batch/api/Azure.Storage.Blobs.Batch.netstandard2.0.cs

```diff
@@ -1,9 +1,17 @@
+namespace Azure.Storage.Blobs.Models
+{
+    public enum BlobDeleteType
+    {
+        Permanent = 0,
+    }
```

Comment:
How's that related to batch?

Reply:
Dumb swagger readme thing, I will remove before merging.

Reply:
fixed.